### ISSUE TITLE
Posenet classification inaccuracy on rotated heads/bodies - need for more diverse training data?

### ISSUE THREAD

**Issue by miktoki** [tensorflow/tfjs#842]
When tilting your head the classification accuracy is drastically reduced to the point where a >90 degree rotated camera will have very poor performance. I realize that that this might seem like a contrived problem, as most people will keep their body upright, and most data augmentation techniques for this type of data will only consider smaller rotations.

This makes it more difficult to estimate non-standard poses, eg. of people doing yoga or swimming. Also simply tilting the head will tend to predict less rotated heads.

I naïvely assume posenet is based on the COCO dataset, which is not primarily targeted at capturing a diverse set of human poses. I suggest including data from other datasets with more diverse human poses (eg. [MPII](http://human-pose.mpi-inf.mpg.de/)) to mitigate this problem.

**Comment by oveddan** [comment]
I think it would definitely be useful to be able to train posenet models targeted towards specific use cases.  Unfortunately the code to train the model has not been open sourced as of now. See #572 and #446

**Comment by nsthorat** [comment]
Closing this out -- this isn't a TensorFlow.js issue but rather one with the PoseNet model itself.
